eslint.config.mjs

```diff
@@ -305,7 +306,7 @@ export default [
     rules: {
       "n/no-unsupported-features/node-builtins": [
         "error",
-        { version: "12.17.0", ignores: ["module"] },
+        { version: "20.19.0", ignores: ["module"] },
```

Comment:
Question about this (both the old and new config): we also run tests in Node.js 6, why don't we use 6 here?